core/domain/stats_jobs_one_off.py

```diff
@@ -34,6 +36,46 @@
 ])
 
 
+class RemoveBadPlaythroughsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """A one-off job for removing all playthroughs created between the July 2018
+    release and the August 2018 release.
+    """
+
+    AUGUST_RELEASE_DATETIME = datetime.datetime(2018, 9, 3, 7, 47, 0, 0)
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [stats_models.ExplorationIssuesModel]
+
+    @staticmethod
+    def map(exp_issues_model):
+        whitelisted_exp_ids = (
+            config_domain.WHITELISTED_EXPLORATION_IDS_FOR_PLAYTHROUGHS.value)
+        if exp_issues_model.exp_id not in whitelisted_exp_ids:
+            return
+        for e_idx, exp_issue in enumerate(exp_issues_model.unresolved_issues):
+            playthroughs_to_remove = []
+            playthrough_instances = stats_models.PlaythroughModel.get_multi(
+                exp_issue['playthrough_ids'])
+            for playthrough_instance in playthrough_instances:
+                if playthrough_instance.created_on < (
+                        RemoveBadPlaythroughsOneOffJob.AUGUST_RELEASE_DATETIME):
+                    playthroughs_to_remove.append(playthrough_instance)
+            stats_models.PlaythroughModel.delete_multi(
+                playthroughs_to_remove)
+            if len(playthroughs_to_remove) == len(exp_issue['playthrough_ids']):
+                exp_issues_model.unresolved_issues.pop(e_idx)
+        exp_issues_model.put()
+        yield(
```

Comment:
For success cases, just yield aggregate counts. Only for error cases, yield individual data.

Less noisy output, that way.